### PR TITLE
Feature/fuel range subscribe unsubscribe vd

### PR DESCRIPTION
**Pull request by GetmanetsIrina**
- invalid types

### CODE REVIEW

File: test_scripts/API/VehicleData/UnsubscribeVehicleData/006_UnsubscribeVD_fuelRange_invalid_type_inResponse.lua [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+-- User story: TO ADD !!!
+-- Use case: TO ADD !!!
+--
+-- Requirement summary:
+-- [SubscribeVehicleData] As a mobile app wants to send a request to unsubscribe for specified parameter
+--
+-- Description:
+-- In case:
+-- 1) mobile application sends valid UnsubscribeVehicleData
+-- 2) HMI responds to SDL with invalid types of fuelRange parameterы
```
Comment (dboltovskyi) [inline test_scripts/API/VehicleData/UnsubscribeVehicleData/006_UnsubscribeVD_fuelRange_invalid_type_inResponse.lua]: @GetmanetsIrina Please remove 'ы' character

File: test_scripts/API/VehicleData/UnsubscribeVehicleData/006_UnsubscribeVD_fuelRange_invalid_type_inResponse.lua [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+--[[ Scenario ]]
+runner.Title("Preconditions")
+runner.Step("Clean environment", common.preconditions)
+runner.Step("Start SDL, HMI, connect Mobile, start Session", common.start)
+runner.Step("RAI with PTU", common.registerAppWithPTU)
+runner.Step("Activate App", common.activateApp)
+runner.Step("SubscribeVehicleData", SubscribeVD)
+
+runner.Title("Test")
+-- runner.Step("SubscribeVehicleData_fuelRange_invaliDataType", SubscribeVDinvalidParamType)
```
Comment (dboltovskyi) [inline test_scripts/API/VehicleData/UnsubscribeVehicleData/006_UnsubscribeVD_fuelRange_invalid_type_inResponse.lua]: @GetmanetsIrina Please remove commented code

File: test_scripts/API/VehicleData/UnsubscribeVehicleData/006_UnsubscribeVD_fuelRange_invalid_type_inResponse.lua (hunk 1 above)
Comment (GetmanetsIrina) [reply test_scripts/API/VehicleData/UnsubscribeVehicleData/006_UnsubscribeVD_fuelRange_invalid_type_inResponse.lua]: Done

File: test_scripts/API/VehicleData/UnsubscribeVehicleData/006_UnsubscribeVD_fuelRange_invalid_type_inResponse.lua (hunk 2 above)
Comment (GetmanetsIrina) [reply test_scripts/API/VehicleData/UnsubscribeVehicleData/006_UnsubscribeVD_fuelRange_invalid_type_inResponse.lua]: Done

**Pull request merged**